salt/roster/flat.py

```diff
@@ -129,7 +130,7 @@ def ret_range_minions(self):
             raise RuntimeError("Python lib 'seco.range' is not available")
 
         minions = {}
-        range_hosts = _convert_range_to_list(self.tgt, __opts__['range_server'])
+        range_hosts = _convert_range_to_list(self.tgt, self.opts['range_server'])
```

Comment:
Just a note: then make sure that `range_server` key is always in the `opts`. Otherwise this will crash.